docs/Tutorials/Scaling/Kubernetes/deployingk8s.md

```diff
@@ -4,15 +4,91 @@ This tutorial demonstrates how to deploy the Open AMT Cloud Toolkit on a local K
 
 Kubernetes, also known as K8s, is an open-source system for automating deployment, scaling, and management of containerized applications.  Learn more about Kubernetes [here](https://kubernetes.io/docs/home/).
 
+
 ## Prerequisites
 
 - [Docker Desktop](https://www.docker.com/products/docker-desktop) with [Kubernetes Enabled](https://docs.docker.com/desktop/kubernetes/)
+
+    !!!important "Important - For Linux"
+        If deploying on a Linux machine, Docker Desktop is not available. You must use Docker Engine alongside a local Kubernetes cluster tool such as [minikube](https://minikube.sigs.k8s.io/docs/) or [kubeadm](https://kubernetes.io/docs/reference/setup-tools/kubeadm/).
+
 - [kubectl](https://kubernetes.io/docs/tasks/tools/)
 - [Helm CLI (v3.5+)](https://helm.sh/)
-- [PSQL CLI (11.13)](https://www.postgresql.org/download/)
-
-!!!important "Important - For Linux"
-    If deploying on a Linux machine, Docker Desktop is not available. You must use Docker Engine alongside a local Kubernetes cluster tool such as [minikube](https://minikube.sigs.k8s.io/docs/) or [kubeadm](https://kubernetes.io/docs/reference/setup-tools/kubeadm/).
+- [Local PostgreSQL server](https://www.postgresql.org/download/) or PostgreSQL Docker Container
+
+    !!!note "Note - Database Required"
+        This guide requires a standalone database for storage. This can be done either as a local Postgres server on your local IP (192.168.XX.X) or as a Docker container.
```

Comment:
Most importantly and recommended is a managed database instance

Reply:
Also 192.168.x.x is to prescriptive, many local networks are 10.x.x.x or 172.x.x.x